meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr-fit/apalis-imx6/boot.cmd

```diff
@@ -12,5 +12,6 @@ setenv rootpart 2
 
 setenv fdt_addr 0x15800000
 setenv optee_ovl_addr 0x16000000
+setenv initrd_addr ${loadaddr}
```

Comment:
Since initrd is not necessarily enough on all targets and we can also load the initrd from the fit into a fixed address, it would be better to have this set as fit_addr or similar.

Reply:
changed to `$fit_addr`

Interesting that I can not use `$loadaddr` for FIT image on imx8mmevk/imx8mqevk, as bootm reports that it's corrupted, something overlaps with that region.

So for now I just do this:
```
setenv fit_addr ${initrd_addr}
```